GitUI/UserControls/FileStatusList.cs

```diff
@@ -802,6 +804,13 @@ private void SelectItems(Func<ListViewItem, bool> predicate)
             StoreNextIndexToSelect();
         }
 
+        private void SetDeleteFilterButtonVisibilityAndFilterComboBoxSize()
```

Comment:
How about?
```suggestion
        private void ToggleFilterControl()
```

Reply:
"Toggle" means "change the state", which is not correct here.
Showing the `DeleteFilterButton` floating as Gerhard requested (for consistency) was much easier than thought. So the function name has simplified.

Reply:
> Showing the `DeleteFilterButton` floating as Gerhard requested

Do screenshots need to be updated?

Reply:
I updated the screenshots yesterday.